src/controllers/submitfeedbackController.js

```diff
@@ -0,0 +1,39 @@
+/* eslint no-underscore-dangle: 0 */
+import FeedbackSubmissionModel from '../models/feedbackSubmission.model';
+
+const submitfeedback = async (req, res) => {
+  const newFeedback = new FeedbackSubmissionModel({
+    feedbackForm_ID: req.body._id,
+    subject: req.body.subject,
+    topic: req.body.topic,
+    subtopics: req.body.subtopics.reduce((acc, curr) => {
+      return [...acc, {
+        subtopicName: curr.subtopicName,
+        rating: curr.rating,
+      },
+      ];
+    }, []),
+    comments: req.body.comments,
+  });
+  try {
+    await newFeedback.save();
+    console.log('feedback submitted');
+    res.json({ success: 'feedback submitted successfully' });
+  } catch (error) {
+    if (error.message.includes('subject: Path `subject` is required.')) {
```

Comment:
Why backticks?

Reply:
backticks are part of the error message from mongoose validator.